health/README.md

```diff
@@ -675,12 +675,12 @@ Check if user or system dimension is using more than 50% of cpu:
     os: linux
 lookup: average -3s percentage foreach system,user
  units: %
- every: 1s
+ every: 10s
   warn: $this > 50
   crit: $this > 80
 ```
 
-The `lookup` line will calculate the average of CPU usage from system and user in the last 3 seconds, due the fact we have
+The lookup line will calculate the average CPU usage from system and user in the last 3 seconds, due to the fact we have
```

Comment:
**lookup** should be an inline code `lookup`:

Add backticks to it, it will look like the previous commit.

Reply:
When I pasted your correction I lost it, thank you for the review, I brought it back now.